docs/learn/learn-elastic-scaling.md

```diff
@@ -25,18 +25,10 @@ respectively. Polkadot also provides throughput boost via parallel computation f
 with **elastic scaling**: parachains can use multiple cores to include multiple parablocks within
 the same relay chain block.
 
-The relay chain receives a sequence of parachain blocks on multiple cores, which are validated and
-checked if all their state roots line up during their inclusion, but assume they’re unrelated
-parachain blocks during backing, availability, and approvals. With elastic scaling implemented, a
-parachain's throughput depends upon its collator infrastructure.
+The capabilities of elastic scaling were tested live on Kusama in December 2024 with [the Spammening Event](https://polkadot.com/spammening/). During this event, 143,343 TPS were recorded on 23 out of 100 cores (23% capacity), consistently achieving an average block time of 6.3 seconds and finality of 16.5 seconds.
 
-The [elastic scaling implementation](https://github.com/paritytech/polkadot-sdk/issues/1829) will be
-rolled out in multiple phases. In the first phase, elastic scaling is set to work on parachains with
-a trusted/permissioned collator set. With this restriction, it is possible to launch elastic scaling
-without changing the candidate receipt. After successfully implementing the first phase, changes can
-be made to the candidate receipt so the collator set can be untrusted/permissionless again. The
-final phase will feature full integration with the Cumulus framework, enabling parachains to be
-configured to access multiple cores continuously.
+!!! info "Polkadot 2.0"
```

Comment:
I think I would leave out "Polkadot 2.0" here, as it seems that the upgrade path is more akin to "Polkadot -> JAM", and maybe it would be better to leave this page as purely elastic scaling-focused.